stories/card/card.stories.js

```diff
@@ -221,7 +221,9 @@ export const analyticalCard = () => `
                             </div>
                             <div class="fd-numeric-content__scale-container">
                                 <div class="fd-numeric-content__scale">
-                                    <span class="fd-numeric-content__scale-arrow sap-icon--down"></span>
+                                    <span class="fd-numeric-content__scale-arrow">
+                                        <i class="sap-icon--down"></i>
```

Comment:
I know this is a draft but just wanted to catch it early before you do too much work - all of these icons will need either aria-hidden or aria-label attributes. aria-hidden if there is already text, aria-label if not (like this example)